### PR TITLE
russian translation

### PR DESCRIPTION
**Pull request by DmitryFisk**

### CODE REVIEW

**Comment by DmitryFisk** [comment]
Oh wait it doesn't even work bruh

**Comment by DmitryFisk** [comment]
Will try to fix now

**Comment by DmitryFisk** [comment]
Built the project on my PC and it told me that the localization file doesn't even exist(???)

**Comment by nomi-san** [comment]
You need to add the .xaml file to the **loader** project or via the **.csproj** file.

File: loader/Languages/ru-RU.xaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<ResourceDictionary
```
Comment (nomi-san) [inline loader/Languages/ru-RU.xaml:1]: Add this file to the project.

**Pull request merged**